releng/org.openhwgroup.corev.ide.target/org.openhwgroup.corev.ide.target.target

```diff
@@ -28,22 +27,37 @@
 			<repository location="https://download.eclipse.org/tools/cdt/builds/9.11/cdt-9.11.1-2020-06-rc1"/>
 		</location>
 		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
-			<unit id="ilg.gnumcueclipse.codered.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.debug.gdbjtag.jlink.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.debug.gdbjtag.jumper.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.debug.gdbjtag.openocd.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.debug.gdbjtag.pyocd.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.debug.gdbjtag.qemu.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.doc.user.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.managedbuild.cross.arm.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.managedbuild.cross.riscv.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.ad.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.cortexm.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.freescale.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.sifive.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.templates.stm.feature.feature.group" version="0.0.0"/>
-			<unit id="ilg.gnumcueclipse.packs.feature.feature.group" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/embed-cdt/releases/5.1.1/p2/"/>
+			<repository location="https://download.eclipse.org/embed-cdt/releases/5.1.3/p2/"/>
```

Comment:
Let's do separate PR if we need to upgrade to this version

Reply:
done